medicinal_plants/lib/src/screens/collection-screen.dart

```diff
@@ -0,0 +1,118 @@
+// ignore_for_file: file_names
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class CollectionScreen extends StatelessWidget {
+  final List<Map<String, String>> items = List<Map<String, String>>.generate(
+    20,
+    (index) => {
+      'image': 'assets/images/Sample plants.png', 
+      'name': 'Monstera',
+      'category': 'Cough',
+    },
+  );
+  CollectionScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+
+    return Scaffold(
+      body: Stack(
```

Comment:
you can use appBar property from Scaffold, instead of using Stack,
and instead of children with GestureDetector and a child of container, 
you can simplify it using ListTile